src/pkg/repository/repository_load_test.go

```diff
@@ -552,3 +560,101 @@ func (suite *RepositoryIndividualLoadTestOneDriveSuite) TestOneDrive() {
 		sel, sel, // same selection for backup and restore
 	)
 }
+
+// ------------------------------------------------------------------------------------------------
+// SharePoint
+// ------------------------------------------------------------------------------------------------
+
+type RepositoryLoadTestSharePointSuite struct {
+	suite.Suite
+	ctx            context.Context
+	repo           repository.Repository
+	acct           account.Account
+	st             storage.Storage
+	sitesUnderTest []string
+}
+
+func TestRepositoryLoadTestSharePointSuite(t *testing.T) {
+	if err := tester.RunOnAny(tester.CorsoLoadTests); err != nil {
+		t.Skip(err)
+	}
+
+	suite.Run(t, new(RepositoryLoadTestSharePointSuite))
+}
+
+func (suite *RepositoryLoadTestSharePointSuite) SetupSuite() {
+	t := suite.T()
+	t.Skip("not running sharepoint load tests atm")
+	t.Parallel()
+	suite.ctx, suite.repo, suite.acct, suite.st = initM365Repo(t)
+	suite.sitesUnderTest = orgSiteSet(t)
+}
+
+func (suite *RepositoryLoadTestSharePointSuite) TeardownSuite() {
+	suite.repo.Close(suite.ctx)
+}
+
+func (suite *RepositoryLoadTestSharePointSuite) TestSharePoint() {
+	ctx, flush := tester.WithContext(suite.ctx)
+	defer flush()
+
+	bsel := selectors.NewSharePointBackup()
+	bsel.Include(bsel.Sites(suite.sitesUnderTest))
+	sel := bsel.Selector
+
+	runLoadTest(
+		suite.T(),
+		ctx,
+		suite.repo,
+		"all_sites", "share_point",
+		suite.sitesUnderTest,
+		sel, sel, // same selection for backup and restore
```

Comment:
is restore defined for SharePoint right now?

Reply:
Nope, but if you look a little up from here you'll see that these tests are currently skipped.